BIPs/2024-W23/BIP-613/inputs.json

```diff
@@ -0,0 +1,123 @@
+[
+  {
+    "chain_map": {
+      "mode": 34443,
+      "fraxtal": 252
+    },
+    "deployments": [
+      "weighted-pool-v4",
+      "composable-stable-pool-v6",
+      "composable-stable-pool-v5",
+      "20230409-erc4626-linear-pool-v4",
+      "20230409-gearbox-linear-pool-v2",
+      "yearn-linear-pool-v2",
+      "20230410-aave-linear-pool-v5",
+      "20230411-managed-pool-v2"
```

Comment:
I'm not sure how this file works but we only deployed to Fraxtal: WeightedV4, ComposableV6, ManagedV2, and LBPs.

Reply:
It will add any permissions it can (if the deployment is there).  Easier to just leave it like this.